pytest.ini

```diff
@@ -14,10 +12,16 @@
 #
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
-# ----------------------------------------------------------------------------
 
 [pytest]
-addopts = -v -s -ra --color=auto --doctest-glob='*.md'
-# Some tests fail when running in a different time zone (e.g. Pacific/Auckland)
-markers = slow
-testpaths = metomi/isodatetime/tests
+addopts =
+    -v
+    -s
+    -ra
+    --color=auto
+    --doctest-glob='*.md'
+    -m 'not slow'
```

Comment:
One job in GH Actions timed out after 45 minutes. Maybe there's something missing here?

Reply:
I'm pretty sure it's just the coverage run being slow